### PR TITLE
Jm/events

### PR DESCRIPTION
**Pull request by judithpatudith**
## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).

### CODE REVIEW

File: src/events.json [hunk 1]
```diff
@@ -655,7 +655,7 @@
   { "title": "Apache Mesos & Artificial Intelligence (Meetup)",
       "image": "/assets/images/events/cluj-napoca.jpg",
       "date": "2018-02-21",
-      "url": "/events/2018/introduction-to-mesos-and-dc-os-cluj"
+      "url": "https://www.meetup.com/Big-Data-Data-Science-Meetup-Cluj-Napoca/events/247378623/"
```
Comment (pleia2) [inline src/events.json]: Why are we restoring the Meetup link in favor of the /events link Matt created?

File: src/events.json [hunk 2]
```diff
@@ -682,24 +682,109 @@
...
+
   { "title": "Operating Flink on Mesos at Scale (Flink Forward)",
        "image": "/assets/images/events/San_Francisco.jpg",
        "date": "2018-04-09",
        "url": "https://sf-2018.flink-forward.org/kb_sessions/operating-flink-on-mesos-at-scale/"
    },
 
+  { "title": "Deep learning beyond the learning (Codemotion World)",
+      "image": "/assets/images/events/Rome.jpg",
+      "date": "2018-04-11",
```
Comment (pleia2) [inline src/events.json]: Looks like this one landed on the 13th specifically https://rome2018.codemotionworld.com/conference/13-April/

File: src/events.json (hunk 1 above)
Comment (judithpatudith) [reply src/events.json]: Because of a merge conflict... Good catch!

Comment (judithpatudith) [reply src/events.json]: Fixing now...

**Comment by pleia2** [comment]
retest this please

**Pull request merged**